src/briefcase/integrations/base.py

```diff
@@ -234,6 +234,22 @@ def system_encoding(self) -> str:
 
         return encoding.upper()
 
+    def unpack_archive_kwargs(self, archive_path: str | os.PathLike) -> dict[str, str]:
```

Comment:
Putting this here is less than ideal...

Like we talked about at PyCon, we should create a `File` tool that would encompass #1826 and subsume the `Download` tool. So, if the `File` tool already existed, I would just put this there.

Furthermore, though, I think we should just wrap the whole archive unpacking process instead of calling `shutil.unpack_archive()` throughout Briefcase.

Given #1826 is still in limbo and I don't want to introduce the `File` tool right now....putting it here seems like a reasonable compromise...

Reply:
Also, I gave up trying to add this `kwargs` to the `shutil.unpack_archive()` calls for Android SDK and Java....the tests were really disagreeing....and those archives are also ZIPs.

Reply:
Moved in to `File` tool